### ISSUE TITLE
Allow user to add custom plugins or options for existing plugins

### ISSUE THREAD

**Issue by boomzero** [Yelp/detect-secrets#291]
I see that your app can detect most common secret incidents in repo. 
But wouldn't it be nice to allow user to add their own patterns for some specific cases?
For example,  `USER_TOKEN="secret"` would be secret in a specific project but not in most of others (it depends on project context) and user can add `USER_TOKEN` to their blacklist word.

**Comment by KevinHock** [comment]
There is an open PR for bring-your-own-plug-ins, but it is blocked on a PR I’ll be making within a week (dropping Python 2 support).

**Comment by boomzero** [comment]
Nice, thank you.

**Comment by nightwatchcyber** [comment]
@KevinHock Assuming this is #255 - any timeline on when that will be merged into master and released?

Thanks!

**Comment by KevinHock** [comment]
cc @OiCMudkips ([reviewer](https://github.com/Yelp/detect-secrets/pull/255#issuecomment-615997373))

**Comment by OiCMudkips** [comment]
This will be released today.

**Comment by nightwatchcyber** [comment]
Thank you!
